src/meta/src/barrier/command.rs

```diff
@@ -84,10 +84,10 @@ pub struct Reschedule {
     pub newly_created_actors: Vec<(StreamActor, PbActorStatus)>,
 }
 
-/// Replacing an old table with a new one. All actors in the table job will be rebuilt.
-/// Used for `ALTER TABLE` ([`Command::ReplaceTable`]) and sink into table ([`Command::CreateStreamingJob`]).
+/// Replacing an old job with a new one. All actors in the job will be rebuilt.
+/// Used for `ALTER TABLE` ([`Command::ReplaceStreamJob`]) and sink into table ([`Command::CreateStreamingJob`]).
 #[derive(Debug, Clone)]
-pub struct ReplaceTablePlan {
+pub struct ReplaceStreamJobPlan {
```

Comment:
We can see no fields here is strongly related with `Table`.

Reply:
On the contrary, the proto contains `catalog.Table`

https://github.com/risingwavelabs/risingwave/blob/11-22-refactor_rename_replace_table_to_replace_stream_job/proto/ddl_service.proto#L356-L369